### ISSUE TITLE
Any solution to minimize bodymovin?

### ISSUE THREAD

**Issue by walleva** [chenqingspring/react-lottie#37]
As bodymovin is huge, could react-lottie add a prop to control which version of bodymovin to use? 
I mean, like 'full', 'min' (bodymovin.min), 'light' (bodymovin_light.min). 
For example, if I only need bodymovin to support svg, do we have a way to require bodymovin_light.min, and make compiled js file as small as possible?
Thanks!

**Comment by urgentMilkMan** [comment]
Yeah adding light would be a serious savings and be far more flexible for adding this.
